website/www/site/data/en/pillars_social.yaml

```diff
@@ -13,9 +13,12 @@
 - name: pillars-github-icon
   icon: icons/github-icon.svg
   url: https://github.com/apache/beam
+- name: pillars-linkedin-icon
+  icon: icons/linkedin-icon.svg
+  url: https://beam.apache.org/
```

Comment:
is the linkedin icon supposed to lead to beam.apache.org?

Reply:
changed